.github/workflows/test.yml

```diff
@@ -78,3 +78,20 @@ jobs:
               uses: codecov/codecov-action@v1.0.10
               with: 
                 yml: ./codecov.yml
+
+    rpcTest:
+        name: RPC Generator Tests
+        runs-on: ubuntu-latest
+        strategy:
+            fail-fast: false
+        steps:
+            - name: Checkout repository
+              uses: actions/checkout@v2.3.1
+              with:
+                  submodules: true
+            - name: Current Directory
+              run: ls
```

Comment:
Is this command necessary or was it only for debug?

Reply:
It is not necessary, i added it for debug